### ISSUE TITLE
A question about channel selecting in formula Z_unknown

### ISSUE THREAD

**Issue by qq237942920** [uber-research/UPSNet#4]
Thanks for your good project, I have a question about your code in detail. The code in your project you released `void_logits = torch.max(fcn_output['fcn_score'][:, (config.dataset.num_classes - 1):, ...], dim=1, keepdim=True)[0] - torch.max(seg_inst_logits, dim=1, keepdim=True)[0] 
` is seems to be not corresponding to the paper formula `Z_unknown = max (X_thing) - max (X_mask)`.
The first max item should be 52:132 channel in fcn_output['fcn_score']
` ([:, (config.dataset.num_seg_classes-config.dataset.num_classes+1):, ...]) ` which represents the things ? I am so sorry for my English and I don't know whether the description of question allows you understand what I want to ask?

**Comment by YuwenXiong** [comment]
Yeah you are right, it should be `(config.dataset.num_seg_classes-config.dataset.num_classes+1)` just like here: https://github.com/uber-research/UPSNet/blob/master/upsnet/operators/modules/unary_logits.py#L73, otherwise for cityscapes it involves extra 3 stuff classes and for coco it removes the first 27 thing classes, which I think will cause performance degradation. I apologize for the mistake in the released code and thanks for your attention and pointing out it! Now it has been fixed.
